test/res.set.js

```diff
@@ -26,24 +26,42 @@ describe('res', function(){
       });
 
       request(app)
-      .get('/')
-      .expect('X-Number', '123')
-      .expect(200, 'string', done);
+        .get('/')
+        .expect('X-Number', '123')
+        .expect(200, 'string', done);
+    })
+
+    it('should raise error if value is not a string for \"content-type\" field', function (done) {
+      var app = express();
+
+      app.use(function (req, res) {
+        try {
```

Comment:
You don't actually need to wrap in `try ... catch`, as all handlers / middleware are already wrapped by Express. Just call `res.set` in the handler directly.